src/pages/Build/CodeOfConductPage.vue

```diff
@@ -0,0 +1,136 @@
+				</p>
+				<h2>6. Be nice to the API servers</h2>
+				<p>
+					Share and share alike. The more often you make calls to the API,
+					the less resources there are available for other developers and the more
+					Kiva's operating expenses rise. Don't request data you don't need.
+					Cache data when possible. Refreshed cached data only when it expires or
+					you hit a cache miss (e.g., you come across an image URL template that
+					is previously unknown to you). If your application is slow because you
+					are making a lot of API calls or the API calls take a long time to return,
+					you're probably doing something wrong. If you think that's not the case,
+					please bring up your situation in the forums so we can help you or modify
+					the API to make what you want to do faster and less demanding on the system.
+				</p>
+				<h2>7. Link images back to Kiva whenever possible</h2>
+				<p>
+					When writing applications that use Kiva images, we'd like you to hyperlink
+					those images back to Kiva — either to a full-size image hosted by Kiva or
+					to a entrepreneur or lender profile as appropriate. If you have a great
+					idea for linking the image to something else in your application, that's
+					okay, we'd just like for you eventually to provide a link to the associated
+					data on Kiva somewhere in the application flow. This helps you communicate
+					copyright of the data back to Kiva.
+				</p>
+				<h2>8. Use app_id in your requests</h2>
+				<p>
+					When making API calls or linking back to Kiva, please pass an App ID
+					in your request. Currently, this isn't required by the Terms Of Service,
+					but consider it a polite way of saying hello and letting us know who's
+					using the API. In the future we hope to return the favor by giving you
+					stats on how your app is doing.
+				</p>
+				<h2>9. Offer your users the ability to donate to Kiva</h2>
+				<p>
+					Kiva relies on the donations we ask for at checkout. Either ask your
+					users for a donation and set it in the request to the basket, or
+					leave it blank so we can set it to our suggested default amount.
+				</p>
+				<h2>10. Keep your data in sync with the API</h2>
+				<p>
+					We understand the need to download Kiva's data and store it locally.
+					However, for regulatory and policy reasons, such data can only be stored
+					for a limited time.
+				</p>
+				<p>
+					If Kiva's data changes, this will be reflected in the API. You must
+					keep local copies of Kiva's data in-sync with the current data available
+					through the API.
+				</p>
+			</div>
+		</div>
+	</www-page>
+</template>
+
+<script>
+import WwwPage from '@/components/WwwFrame/WwwPage';
+import DeveloperSecondaryMenu from '@/components/WwwFrame/Menus/DeveloperSecondaryMenu';
+
+export default {
+	components: {
+		WwwPage,
+		DeveloperSecondaryMenu,
+	},
+	metaInfo: {
+		title: 'Code of Conduct'
+	}
+};
+</script>
+
+<style lang="scss">
+@import 'settings';
```

Comment:
You'll only need this if you are using variable in the same style element. Not a biggie.

Reply:
Sure thing. But it's a good idea to remove Line 131 to keep the file clean?